src/test/scala/zio/prelude/AssociativeBothSpec.scala

```diff
@@ -5,7 +5,7 @@ import zio.test.laws._
 
 object AssociativeBothSpec extends DefaultRunnableSpec {
 
-  def spec =
+  def spec: ZSpec[Environment, Failure] =
```

Comment:
Understand the principle of including return types for public members but normally we would not include the return type for `spec` here.

Reply:
That's a reasonable request. Sadly, I don't know how to configure Scalafix in such a way, so that it, for example, omits the `ExplicitResultTypes` rule in `Test` configuration :disappointed:

Reply:
Okay.